SPECS/leveldb/leveldb.spec

```diff
@@ -0,0 +1,208 @@
+%undefine __cmake_in_source_build
+
+Summary:        A fast and lightweight key/value database library by Google
+Name:           leveldb
+Version:        1.22
+Release:        3%{?dist}
+License:        BSD
+URL:            https://github.com/google/leveldb
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+
+# available in https://github.com/fusesource/leveldbjni/blob/leveldb.patch
+Patch0001:      0001-Allow-leveldbjni-build.patch
+# https://github.com/fusesource/leveldbjni/issues/34
+# https://code.google.com/p/leveldb/issues/detail?id=184
+# Add DB::SuspendCompactions() and DB:: ResumeCompactions() methods
+Patch0002:      0002-Added-a-DB-SuspendCompations-and-DB-ResumeCompaction.patch
+# Cherry-picked from Basho's fork
+Patch0003:      0003-allow-Get-calls-to-avoid-copies-into-std-string.patch
+# https://groups.google.com/d/topic/leveldb/SbVPvl4j4vU/discussion
+Patch0004:      0004-bloom_test-failure-on-big-endian-archs.patch
+
+BuildRequires:  cmake
+#Provided by gcc package
+#BuildRequires:  gcc-c++
```

Comment:
Please remove this and similar comments.

Reply:
Done